app/src/main/java/org/simple/clinic/storage/Migration_52_53.kt

```diff
@@ -0,0 +1,15 @@
+package org.simple.clinic.storage
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+class Migration_52_53 : Migration(52, 53) {
+  override fun migrate(database: SupportSQLiteDatabase) {
+    with(database) {
+      inTransaction {
+        execSQL("ALTER TABLE PatientAddress ADD COLUMN houseAddress TEXT DEFAULT NULL")
```

Comment:
We've been using the convention of setting these as string templates (triple-quoted), and quoting the identifiers in the SQL statement as per the SQL conventions. Can we follow the convention please?

```kotlin
execSQL("""ALTER TABLE "PatientAddress" ADD COLUMN "houseAddress" TEXT DEFAULT NULL""")
execSQL("""ALTER TABLE "PatientAddress" ADD COLUMN "zone" TEXT DEFAULT NULL""")
```

Reply:
Done